target/linux/lantiq/ase/base-files/etc/board.d/02_network

```diff
@@ -35,7 +34,7 @@ esac
 ls /lib/modules/$(uname -r)/ltq_atm* 1> /dev/null 2>&1 && \
 	ucidef_add_atm_bridge 1 32 "llc" "bridged" "dsl"
 
-if lantiq_is_vdsl_system; then
+if grep -qE "system type.*: (VR9|xRX200)" /proc/cpuinfo; then
```

Comment:
This looks to me like it could be subtarget-specific, and thus the if might be removed with subtarget-specific files now?
Can somebody assist me on that one?